--- Log opened Sun Sep 30 00:00:17 2012
-!- blackburn [~blackburn@188.168.2.187] has quit [Quit: Leaving.]
 shogun-buildbot	build #118 of nightly_default is complete: Failure [failed test]  Build details are at http://www.shogun-toolbox.org/buildbot/builders/nightly_default/builds/118
-!- naywhayare [~ryan@spoon.lugatgt.org] has quit [Ping timeout: 240 seconds]
-!- naywhayare [~ryan@spoon.lugatgt.org] has joined #shogun
-!- Arya [05c7817b@gateway/web/freenode/ip.5.199.129.123] has joined #shogun
 Arya	Hi,
 Arya	I am getting  "terminate called after throwing an instance of 'shogun::ShogunException'" in libsvm multiclass classification
 Arya	actually, in apply() routine
-!- Arya [05c7817b@gateway/web/freenode/ip.5.199.129.123] has quit [Ping timeout: 245 seconds]
-!- n4nd0 [~nando@s83-179-44-135.cust.tele2.se] has joined #shogun
-!- blackburn [~blackburn@37.61.179.92] has joined #shogun
-!- blackburn [~blackburn@37.61.179.92] has quit [Quit: Leaving.]
-!- n4nd0 [~nando@s83-179-44-135.cust.tele2.se] has quit [Quit: leaving]
--- Log closed Mon Oct 01 00:00:17 2012
